src/rp2/in_transaction.py

```diff
@@ -63,8 +72,22 @@ def __init__(
             self.__crypto_in = configuration.type_check_positive_decimal("crypto_in", crypto_in, non_zero=True)
         self.__crypto_fee: RP2Decimal = configuration.type_check_positive_decimal("crypto_fee", crypto_fee) if crypto_fee else ZERO
         self.__fiat_fee: RP2Decimal = configuration.type_check_positive_decimal("fiat_fee", fiat_fee) if fiat_fee else ZERO
+
+        # After a transfer, the per-wallet application model is populated with artificial InTransactions that model the "to" part
+        # of the transfer (these artificial InTransactions are not present in the universal application model). These fields are
+        # only populated in the artificial InTransactions of the per-wallet application model field and describes where the funds
+        # come from (due to the transfer) and go to (after the transfer).
```

Comment:
A bit of a nitpick, but technically I think this comment should go below `self.__from_lot`